main.sh

```diff
@@ -186,11 +182,19 @@ while read -r __b3bp_tmp_line; do
 
     if [ "${!__b3bp_tmp_varname}" = "1" ]; then
       __b3bp_tmp_init="${__b3bp_tmp_line##*Default=}"
+      __b3bp_tmp_re='^"(.*)"$'
+      if [[ "${__b3bp_tmp_init}" =~ ${__b3bp_tmp_re} ]]; then
+        __b3bp_tmp_init="${BASH_REMATCH[1]}"
+      else
+        __b3bp_tmp_re="^'(.*)'$"
+        if [[ "${__b3bp_tmp_init}" =~ ${__b3bp_tmp_re} ]]; then
+	  __b3bp_tmp_init="${BASH_REMATCH[1]}"
```

Comment:
Are these changes so that you can include quotation marks (single or double) in your default values?